script/Arcueid/Day 9/09_00_ARC09_2_2.txt

```diff
@@ -144,11 +144,12 @@ And besides, look.
 [sha:f102a5486a307353a5970cd75747cb4eeb8d808d]{
 -- Page 82, Offset 21517.
 -- 「うん、鞄に入れてきたものだけで十分。
-"Nope, the things I put in my bag are all I need.
+"Nope, the things I put in my bag werw all I needed.
 }
 [sha:8c629cec38a5f4465c5a53b2c5548018a479a8aa]{
 -- Page 82, Offset 21518.
 -- 大事なものは、ここに来る時に全部持ってきた」
+//There is an arguement to be made for held dear but I think it's fine as is
```

Comment:
Cool. When the time comes to merge, be sure to go back and clean up any comments that are specific to this review, just so they don't confuse use later.